tls/s2n_handshake_io.c

```diff
@@ -651,6 +652,53 @@ static int s2n_conn_update_handshake_hashes(struct s2n_connection *conn, struct
     return 0;
 }
 
+/* this hook runs before hashes are updated */
+static int s2n_conn_pre_handshake_hashes_update(struct s2n_connection *conn)
+{
+    if (conn->actual_protocol_version < S2N_TLS13) return 0;
```

Comment:
We try pretty hard to use braces even when they're not needed, just to avoid problems in slipping over time, like if someone puts in a macro, and then that macro is a multi-statement one.